examples/api/lib/widgets/sliver/pinned_header_sliver.0.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [PinnedHeaderSliver].
+
+void main() {
+  runApp(const PinnedHeaderSliverApp());
+}
+
+class PinnedHeaderSliverApp extends StatelessWidget {
+  const PinnedHeaderSliverApp({ super.key });
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: PinnedHeaderSliverExample(),
+    );
+  }
+}
+
+class PinnedHeaderSliverExample extends StatefulWidget {
+  const PinnedHeaderSliverExample({ super.key });
+
+  @override
+  State<PinnedHeaderSliverExample> createState() => _PinnedHeaderSliverExampleState();
+}
+
+class _PinnedHeaderSliverExampleState extends State<PinnedHeaderSliverExample> {
+  int count = 0;
+  late final ScrollController scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    scrollController = ScrollController();
+  }
+
+  @override
+  void dispose() {
+    scrollController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final ColorScheme colorScheme = theme.colorScheme;
+
+    final Widget header = Container(
+      color: colorScheme.background,
+      padding: const EdgeInsets.all(4),
+      child: Material(
+        color: colorScheme.primaryContainer,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(8),
+          side: BorderSide(
+            width: 7,
+            color: colorScheme.outline,
+          ),
+        ),
+        child: Container(
+          alignment: Alignment.center,
+          padding: const EdgeInsets.symmetric(vertical: 48),
+          child: Text(
+            count.isOdd ? 'Alternative Title\nWith Two Lines' : 'PinnedHeaderSliver',
+            style: theme.textTheme.headlineMedium!.copyWith(
+              color: colorScheme.onPrimaryContainer,
+            ),
+          ),
+        ),
+      ),
+    );
+
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 4),
+          child: Scrollbar(
```

Comment:
I forgot to check in the resizing sliver PR, should we remove this scrollbar? When viewed in the API docs on-web-on-desktop, the auto scrollbar kicks in, so there would be two.

Reply:
Good point. I've removed the scrollbar.